packages/material-ui/test/integration/TableCell.test.js

```diff
@@ -79,31 +78,4 @@ describe('<TableRow> integration', () => {
 
     expect(getByTestId('cell')).to.have.class(classes.footer);
   });
-
-  it('sets role="columnheader" when "component" prop is set and used in the context of table head', () => {
-    const { getByTestId } = render(
-      <TableHead component="div">
-        <TableCell component="div" data-testid="cell" />,
-      </TableHead>,
-    );
-    expect(getByTestId('cell')).to.have.attribute('role', 'columnheader');
-  });
-
-  it('sets role="cell" when "component" prop is set and used in the context of table body ', () => {
-    const { getByTestId } = render(
-      <TableBody component="div">
-        <TableCell component="div" data-testid="cell" />,
-      </TableBody>,
-    );
-    expect(getByTestId('cell')).to.have.attribute('role', 'cell');
-  });
-
-  it('sets role="cell" when "component" prop is set and used in the context of table footer ', () => {
-    const { getByTestId } = render(
-      <TableFooter component="div">
-        <TableCell component="div" data-testid="cell" />,
-      </TableFooter>,
-    );
-    expect(getByTestId('cell')).to.have.attribute('role', 'cell');
-  });
```

Comment:
These should stay and document that the role is now incorrect.

Reply:
Thanks for the feedback @eps1lon. I can keep them and make them as negative test